src/booleans.js

```diff
@@ -1,59 +1,61 @@
 export const negate = (a) => {
-  // your code here
+  return !a
 };
 
 export const both = (a, b) => {
-  // your code here
+  return a && b
 };
 
 export const either = (a, b) => {
-  // your code here
+  return a || b
 };
 
 export const none = (a, b) => {
-  // your code here
+  return !a && !b
 };
 
 export const one = (a, b) => {
-  // your code here
+  return !a && b || a && !b
 };
 
 export const truthiness = (a) => {
-  // your code here
+  return Boolean (a)
 };
 
 export const isEqual = (a, b) => {
-  // your code here
+  return a === b
 };
 
 export const isGreaterThan = (a, b) => {
-  // your code here
+  return a > b
 };
 
 export const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  return a <= b
 };
 
 export const isOdd = (a) => {
-  // your code here
+  return a % 2 != 0 
 };
 
 export const isEven = (a) => {
-  // your code here
+  return a % 2 === 0
 };
 
 export const isSquare = (a) => {
-  // your code here
-};
+  return Number.isInteger(Math.sqrt(a))
+};  
 
 export const startsWith = (char, string) => {
-  // your code here
+  return string.startsWith(char)
 };
 
 export const containsVowels = (string) => {
-  // your code here
+  let regex1 = RegExp(/[AEIOUaeiou]/)
```

Comment:
Not sure about the `regex1` variable name. Perhaps `vowelRegex`? Also, this can probably be a constant, as it isn't being reassigned to :).